ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/claimantresponse/CCDResponseAcceptation.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.cmc.ccd.domain.claimantresponse;
+
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.hmcts.cmc.ccd.domain.response.CCDPaymentIntention;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+@Value
+public class CCDResponseAcceptation extends CCDClaimantResponse {
```

Comment:
there is no  @EqualsAndHashCode for this class but parent has

Reply:
added